src/scale.ts

```diff
@@ -180,10 +180,33 @@ export interface ScaleConfig<ES extends ExprRef | SignalRef> {
    */
   clamp?: boolean | ES;
 
+
+  /**
+   * Default inner padding for `x` and `y` band scales with nested `xOffset` and `yOffset` encoding.
+   *
+   * __Default value:__ `0.2`
+   *
+   * @minimum 0
+   * @maximum 1
+   */
+   nestedOffsetPaddingInner?: number | ES;
```

Comment:
Any opinion about this config name?

Reply:
Why is it `nested` and not just offset padding?

Reply:
renamed to  `bandWithNestedOffsetPadding`   --> This is padding of the x/y scale, when they have x/yOffset nested, but the padding is for x/y scales, not x/yOffset scales.  (offsetPaddingInner/Outer is then for x/yOffset scales)